### PR TITLE
chore(gui): Separate quick open actions

### PR DESCRIPTION
**Pull request by JustBuddy**
As mentioned here https://github.com/vrc-get/vrc-get/discussions/894#discussioncomment-10487779, this PR makes the quicklinks used to open static data a separate box, and shortens the localization strings.

image

Adds keys:
"settings:files and directories"
"settings:files and directories:description"

Moves/Changes keys:
"settings:button:open settings.json": "settings.json",
"settings:button:open gui config.json": "gui-config.json",
"settings:button:open logs": "Log folder",
 "settings:button:open custom templates": "Custom Templates Folder",

### CODE REVIEW

**Comment by JustBuddy** [comment]
Not sure if this warrants a Changelog entry, let me know if I should add one.

**Comment by JustBuddy** [comment]
I changed the key positions in some other languages that follow the EN template, I intentionally left the new keys out to make sure the github action runs and notifies translators.

**Comment by JustBuddy** [comment]
Changed wording from "folders" to "directories" to comply with contribution standards.

File: vrc-get-gui/locales/en.json5 [hunk 1]
```diff
@@ -391,12 +391,15 @@
     "settings:theme:light": "Light",
     "settings:theme:dark": "Dark",
 
+    "settings:files and directories": "Files & Directories",
+    "settings:files and directories:description": "Quickly access files and directories at static locations.",
```
Comment (anatawa12) [inline vrc-get-gui/locales/en.json5:395]: I think static is not important and we may add direct link to log file for current launch in the future so something like "ALCOM related files and directories" is better.

Comment (JustBuddy) [reply vrc-get-gui/locales/en.json5]: Sure, I adjusted it. Please take a look ^^ @anatawa12

Comment (anatawa12) [reply vrc-get-gui/locales/en.json5]: LGTM! Thank you!

**Comment by JustBuddy** [comment]
Not going to do any changes to this for now, can be merged whenever you see fit.

**Comment by anatawa12** [comment]
I did not merge immediately because just I did not checked the behavior. I checked now and LGTM! Thank you!

**Pull request merged**